docs/reference/commandline/service_create.md

```diff
@@ -384,9 +402,24 @@ volumes in a service:
      </p>
     </td>
   </tr>
+  <tr>
+    <td><b>bind-nonrecursive</b></td>
+    <td>
+      By default, submounts are recursively bind-mounted as well. However, this behavior can be confusing when a
+      bind mount is configured with <tt>readonly</tt> option, because submounts are not mounted as read-only.
+      Set <tt>bind-nonrecursive</tt> to disable recursive bind-mount.<br />
+
```

Comment:
I think you need to remove the blank line here (and below), otherwise the nested HTML is not rendered as HTML (embedding HTML in Markdown is tricky 😞). Here's how it renders currently:

<img width="972" alt="screen shot 2019-01-09 at 16 24 22" src="https://user-images.githubusercontent.com/1804568/50909020-3b25e800-142b-11e9-9c3e-69d41590dccc.png">


(Perhaps it's only an issue on GitHub and not on docs.docker.com)